MaterialDesignExtensions/Commands/Internal/AutocompleteCommands.cs

```diff
@@ -1,9 +1,4 @@
-﻿using System;
```

Comment:
I know that there are unused using directives, but removing all of them in this pull request blows the whole thing up. There are hundreds of files changed, but only a few of them have changes specific for the feature mentioned in the title.